packages/imperative/__tests__/plugins/utilities/npm-interface/install.test.ts

```diff
@@ -9,6 +9,7 @@
 *
 */
 
+/* eslint-disable jest/expect-expect */
```

Comment:
we might be able to rename the functions (like `wasNpmInstallCallValid`) which contain the expects to something like `expectNpmInstallCallValid` (or soemthing to that matches the `assertFunctionNames`) 😋